### DISCUSSION TITLE
direction assignment in layups

### DISCUSSION THREAD

**Discussion by sange000000** [wenbinyugroup/.github#25] [VABS Related Discussions]
I see that many  times this right direction assignment is not strictly followed, and layup are done on the left? Is there any limitation on specifying the direction
![image](https://github.com/user-attachments/assets/7ddf1754-4369-4924-ac1a-378205b8cf4b)

**Comment by unitasium** [comment]
Could you show me the complete input of this `component` block?

**Reply by sange000000** [reply]
![image](https://github.com/user-attachments/assets/72947bd5-c126-400d-a350-c4def3a28d8e)

**Reply by sange000000** [reply]
I have still not got a solution for this issue. when I have multiple layups over a line, the direction assignment fails
